common/test/acceptance/pages/studio/course_info.py

```diff
@@ -1,16 +1,169 @@
 """
 Course Updates page.
 """
-
 from common.test.acceptance.pages.studio.course_page import CoursePage
+from common.test.acceptance.pages.studio.utils import type_in_codemirror, set_input_value
 
 
 class CourseUpdatesPage(CoursePage):
     """
     Course Updates page.
     """
-
     url_path = "course_info"
 
     def is_browser_on_page(self):
-        return self.q(css='body.view-updates').present
+        """
+        Returns whether or not the browser on the page and has loaded the required content
+        """
+        return self.q(css='.handouts-content').present
+
+    def are_course_updates_on_page(self):
+        """
+        Is the course updates content Div loaded on the page
+        """
+        return self.q(css='article#course-update-view.course-updates').present
+
+    def is_new_update_button_present(self):
+        """
+        Checks for the presence of the new update post button
+        """
+        return self.q(css='.new-update-button').present
+
+    def click_new_update_button(self):
+        """
+        Wait for the handouts-content, otherwise the render of the page will clear the new update
+        form when loading content
+        """
+        self.wait_for_element_presence('.handouts-content', '.handouts-content')
+        self.q(css='.new-update-button').click()
+        self.wait_for_page()
+        self.wait_for_element_visibility('.CodeMirror', 'Waiting for .CodeMirror')
+
+    def is_new_update_form_present(self):
+        """
+        Has the new update form loaded and it is present
+        """
+        return self.q(css='.CodeMirror').present
+
+    def submit_update(self, message):
+        """
+        Adds update text to the new update CodeMirror form and submits that text
+
+        Arguments:
+            message (str): The message to be added and saved
+        """
+        type_in_codemirror(self, 0, message)
+        if self.is_new_update_save_button_present():
+            self.click_new_update_save_button()
+            self.wait_for_page()
+
+    def set_date(self, date):
+        """
+        Sets the updates date input to the provided value
+
+        Arguments:
+            date (str): Date string in the format DD/MM/YYYY
+        """
+        set_input_value(self, 'input.date', date)
+
+    def is_update_date(self, search_date):
+        """
+        Checks to see if the search date is present
+
+        Arguments:
+            search_date (str): e.g. 06/01/2013 would be found with June 1, 2013
+        """
+        dates = self.q(css='.date-display').html
```

Comment:
This method will return on the first match, I was attempting to make it a little more versatile, but I can simplify to just this use.

Currently if there is more than one update I will get a list of dates and then I will check all of them.